### PR TITLE
Adds the labs: best practices page

### PR DESCRIPTION
**Pull request by parrish**
Currently deployed to https://preview.zooniverse.org/panoptes-front-end/best-practices/

Pending review from @jwbmartin

### CODE REVIEW

**Comment by jwbmartin** [comment]
That'll do.

File: app/pages/lab/best-practices/great-project.cjsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  displayName: 'LabBestPracticesGreatProjectsPage'
+
+  componentDidMount: ->
+    document.documentElement.classList.add 'on-secondary-page'
+
+  componentWillUnmount: ->
+    document.documentElement.classList.remove 'on-secondary-page'
+
+  render: ->
+    <Markdown>{counterpart "bestPracticesGreatProjectsPage.content"}</Markdown>
```
Comment (brian-c) [inline app/pages/lab/best-practices/great-project.cjsx]: Should this `counterpart` call be a `<Translate />` component?

Comment (parrish) [inline app/pages/lab/best-practices/great-project.cjsx]: Maybe? How would you suggest combining a `<Markdown />` component with a  `<Translate />` component?

Comment (parrish) [inline app/pages/lab/best-practices/great-project.cjsx]: For reference, this is how the [about pages](https://github.com/zooniverse/Panoptes-Front-End/blob/master/app/pages/about/about-home.cjsx#L47) are set up.

Comment (brian-c) [inline app/pages/lab/best-practices/great-project.cjsx]: Ha, excellent point. Looks good then.

**Pull request merged**